pkg/capacityscheduling/capacity_scheduling.go

```diff
@@ -188,7 +188,7 @@ func New(obj runtime.Object, handle framework.Handle) (framework.Plugin, error)
 func (c *CapacityScheduling) EventsToRegister() []framework.ClusterEvent {
 	// To register a custom event, follow the naming convention at:
 	// https://git.k8s.io/kubernetes/pkg/scheduler/eventhandlers.go#L403-L410
-	eqGVK := fmt.Sprintf("elasticquotas.v1alpha1.%v", scheduling.GroupName)
+	eqGVK := fmt.Sprintf("elasticquotas.v1alpha1.%v", scheduling.SchemaGroupName)
```

Comment:
Let's get rid of the duplicated GroupName variable definitions. Just unify the usage to refer to the k/k variable.